File: dayTwo/Java/Level2/TaskFive.java
import java.util.Scanner;

public class TaskFive {

public static void main(String[] args) {

Scanner userInput = new Scanner(System.in);

System.out.print("Enter first number ");

int firstNumber = userInput.nextInt();

System.out.print("Enter second number ");

int secondNumber = userInput.nextInt();

System.out.print("Enter third number ");

int thirdNumber = userInput.nextInt();

if(firstNumber > secondNumber) {

System.out.println(firstNumber);
}else if (secondNumber > thirdNumber){
System.out.println(secondNumber);
}else if (thirdNumber > firstNumber){

System.out.println(thirdNumber);
}

}

}

File: dayTwo/Python/Level2/taskfive.py
first_number = float(input(" Enter first number: "))
second_number = float(input("   Enter second number: "))
third_number = float(input("   Enter third number: "))
if first_number >= second_number and first_number >= third_number:
    largest = first_number
elif second_number >= first_number and second_number >= first_number:
    largest = second_number
else:
    largest = third_number
print(f"Largest number: {largest}"
